docs/pages/guides/using-custom-fonts.md

```diff
@@ -44,7 +43,41 @@ export default function App() {
 
 To create a new project including this example, run `npx create-react-native-app --template with-custom-font` in your terminal.
 
-<SnackEmbed snackId="@ijzerenhein/custom-font-example" />
+
+<SnackInline label="Custom Font" dependencies={['expo-font']}>
```

Comment:
```suggestion
<SnackInline
  label="Custom Font"
  dependencies={["expo-font"]}
  files={{
    "assets/fonts/Inter-Black.otf":
      "https://snack-code-uploads.s3.us-west-1.amazonaws.com/~asset/44b1541a96341780b29112665c66ac67",
  }}
>
```

what do you think about using props to SnackInline instead? to me this is simpler (don't need to generate an anchor tag / parse it out) and it's consistent with how we currently specify dependencies as well

Reply:
I agree with that. I tried the anchors as I initially wanted to point to static files on the docs site itself and this would provide link checking and autocomplete.

What would you say of using the prop-name `assets`? Files is more generic and I don't really see how we could support code files in that format.

Reply:
yep that seems better

Reply:
I've updated the PR and added the `assets` prop 👍

Reply:
EDIT: I've changed the prop back to `files`. Because of #9249 and #9278